hw12_13_14_15_calendar/deployments/docker-compose.yml

```diff
@@ -0,0 +1,85 @@
+version: "2.4"
```

Comment:
Used this version of docker-compose to have an ability to use `healthcheck` together with `depends_on`